### DISCUSSION TITLE
How do I add comments to variables?

### DISCUSSION THREAD

**Discussion by Chise1** [fastapi/fastapi#7684] [Questions] [answered]
How do I add comments to variables?
Like this:

![1585038641(1)](https://user-images.githubusercontent.com/47511274/77404563-e5015280-6dec-11ea-9854-65823a66a96f.png)

**Comment by Dustyposa** [comment, marked as answer]
This [one](https://fastapi.tiangolo.com/tutorial/body-fields/) can help you ?
You can tests it and you can see description in swagger page.

**Comment by tiangolo** [comment]
Thanks for your help here @Dustyposa ! :bowing_man: :cake:
